src/encoded/types/biosample.py

```diff
@@ -729,15 +724,15 @@ def generate_summary_dictionary(
     if originated_from_object is not None:
         if 'biosample_term_name' in originated_from_object:
             dict_of_phrases['originated_from'] = ('originated from ' +
-                                                    originated_from_object['biosample_term_name'])
+                                                  originated_from_object['biosample_term_name'])
```

Comment:
We shouldn't make changes not related to the feature.

Reply:
ok